runtime/interpreter/value.go

```diff
@@ -680,6 +683,8 @@ func NewDeferredArrayValue(
 	version uint16,
 ) *ArrayValue {
 	return &ArrayValue{
+		// TODO: type
```

Comment:
Looks like we missed a TODO here. Could you please resolve it?

Reply:
Ah right! We can simply remove the TODO (+ field initializing). Static type is populated when the content is loaded to memory, so we dont need to provide it at the constructor. Same for dictionary as well.

Reply:
Ah, right! 👍

Reply:
Removed it with a comment: 5b2c6ce

Reply:
Thank you for resolving them and also adding comments! 👌